### DISCUSSION TITLE
Migrating Data from FlySpray to GitHub

### DISCUSSION THREAD

**Discussion by wmirza62** [community/community#39444] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

How do I backup and migrate data from FlySpray to GitHub!

**Comment by ghostinhershell** [comment]
Hi @wmirza62,

Thanks for being a part of the GitHub Community, we're glad you're here!

If you're looking for help for this specific topic, you might want to try asking for help somewhere that focuses on this project, such as the [flyspray](https://github.com/flyspray/flyspray) repo. It's possible that another GitHub user might have run into this same issue and can help, but the GitHub Community on Discussions focuses primarily on topics related to GitHub itself or collaboration on project development and ideas. We want to make sure you’re getting the best support you can, but this space may not be the right place for this particular topic.

Best of luck!
